### DISCUSSION TITLE
itwin.js  support split screen display now?

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#682] [Q&A] [answered]
Hi,itwin.js  support split screen display now?

For example, one screen shows a 3D model and the other shows a 2D drawing.
Thanks.

**Comment by roluk** [comment, marked as answer]
Yes, multiple viewports are supported. There's [Cross-Probing sample](https://www.itwinjs.org/sample-showcase/?group=Viewer+Features&sample=cross-probing-sample&imodel=Bay+Town+Process+Plant) that demonstrates this use case.

**Reply by shenshengyi** [reply]
> Yes, multiple viewports are supported. There's [Cross-Probing sample](https://www.itwinjs.org/sample-showcase/?group=Viewer+Features&sample=cross-probing-sample&imodel=Bay+Town+Process+Plant) that demonstrates this use case.

Hi, I mean having two computer monitors. I want 2D drawings to be displayed on one monitor and 3D models to be displayed on the other monitor. Is that supported? thank you.

**Reply by pmconne** [reply]
All a viewport requires is a div. You can have as many viewports as you can divs, and can position them using CSS however you desire. Whether that allows you to control which monitors display which viewports depends entirely on the CSS and HTML specs, not on iModel.js.

In general you don't want to design a UI that *requires* the user to have two monitors though.

**Reply by shenshengyi** [reply]
> All a viewport requires is a div. You can have as many viewports as you can divs, and can position them using CSS however you desire. Whether that allows you to control which monitors display which viewports depends entirely on the CSS and HTML specs, not on iModel.js.
> 
> In general you don't want to design a UI that _requires_ the user to have two monitors though.

Thank you very much.
